client/web/src/enterprise/batches/create/ConfigurationForm.module.scss

```
@@ -4,6 +4,7 @@
     margin: 2rem auto;
     padding-bottom: 3rem;
     width: 50%;
+    max-width: 720px;
```

Honestly maybe just:

```suggestion
    max-width: 720px;
```

😅 Just because it looks a little weird that as your screen scales up, it goes big -> small -> big:


https://user-images.githubusercontent.com/8942601/235394965-cee66936-5655-4786-9189-df951639dca0.mov

---

Oh, that makes sense, then. Funny, I didn't get that janky experience until I increased my screen resolution.
Thanks for testing this.